demo/src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
Not really needed, correct? Cannot see it on https://create-react-app.dev/docs/adding-typescript/

Reply:
This type definition file gets created when you do `npx create-react-app my-app --template typescript`, we can remove it though, up to you!

Reply:
Yep, I saw myself that it's needed when I removed it :D